rum/getting-started.mdx

```diff
@@ -59,6 +59,75 @@ if(window.Middleware){
 
 Once RUM is configured, metrics should begin appearing on the RUM Dashboard in Middleware within a few minutes. For more information on the RUM Dashboards see [here](https://docs.middleware.io/docs/rum/rum-middleware).
 
-For any issues please reach out to our team in your Slack channel or through the help menu in your [Middleware Account](https://app.middleware.io/auth/login).
+# Using RUM
```

Comment:
Put this entire section of new stuff in a doc in Workflows called "Real User Monitoring", and link between the two pages. E.g. the above sentence's hyperlink can be the workflows doc, and from the workflows page you can link back to this one.